client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/QueryDSLDocumentationTests.java

```diff
@@ -465,4 +465,10 @@ public void testRankFeatureSigmoid() {
             0.6f
         );
     }
+
+    public void testRankFeatureLinear() {
```

Comment:
I just realized -- do we need these tests? This code doesn't have tags, so I assume it's not imported into docs?

Reply:
Good catch! It looks like the whole file is not used at all, since we removed transport client docs [in] (https://github.com/elastic/elasticsearch/pull/42483). I will create a separate PR to remove this file.